openedx/core/djangoapps/programs/tasks.py

```diff
@@ -247,8 +243,8 @@ def _retry_with_custom_exception(username, reason, countdown):
             except HTTPError as exc:
                 if exc.response.status_code == 404:
                     LOGGER.exception(
-                        f"Certificate for program {program_uuid} could not be found. " +
-                        f"Unable to award certificate to user {username}. The program might not be configured."
+                        f"Certificate for program {program_uuid} could not be found. "
+                        + f"Unable to award certificate to user {username}. The program might not be configured."
```

Comment:
🤨 I know this wasn't you... but do we even need that concatenation operator? I thought f-strings took care of that for us automagically?